database/dml/postgres/service.go

```diff
@@ -29,6 +29,14 @@ OFFSET $3;
 SELECT count(*) as count
 FROM services
 WHERE build_id = $1
+`
+
+	// SelectServiceImagesCount represents a query to select
+	// the count of an images appearances in the database.
+	SelectServiceImagesCount = `
+SELECT image, COUNT(image)
```

Comment:
I wonder if we want to add the `as count` after the `COUNT(image)` part, similar to how we do in other spots?

i.e. https://github.com/go-vela/server/pull/52/files#diff-da7d2b57765443dd3c6706913bd9b043R29

Also, do you think we should lowercase the `COUNT` here similar to how we do in our other count queries?

i.e. https://github.com/go-vela/server/pull/52/files#diff-da7d2b57765443dd3c6706913bd9b043R29